provider/cloudflare/cloudflare.go

```diff
@@ -348,6 +366,23 @@ func (p *CloudFlareProvider) AdjustEndpoints(endpoints []*endpoint.Endpoint) []*
 	return adjustedEndpoints
 }
 
+func (p *CloudFlareProvider) enhanceEndpoint(record *endpoint.Endpoint, zoneNameIDMapper provider.ZoneIDName, idToZoneMapper map[string]cloudflare.Zone) error {
```

Comment:
Can you find a better name for this method? "Enhance" means nothing and it's not clear what the method does.

Reply:
I've changed it to `addPlanInfoToEndpoint`, which reflects better the method's purpose